src/test/java/steps/LoginSteps.java

```diff
@@ -0,0 +1,48 @@
+package steps;
+
+import AutomationPractice.ui.pages.LoginPage;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.testng.Assert;
+
+/**
+ * LoginSteps class, here are implemented the steps for the login of users.
+ *
+ * @author Maday Alcala
+ * @version 0.0.1
+ */
+public class LoginSteps {
+    private static  LoginPage loginPage;
+
+    /**
+     * This method is in charge of opening the page.
+     */
+    @Given("I visit the login page")
+    public void initializeLoginPage() {
+        loginPage = new LoginPage();
```

Comment:
in this step you should call your transporter and navigate to the login page

Reply:
Done.